psutil/arch/aix/psutil_aix_common.c

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
I would just call this file `common.c`. The `aix` bit is redundant because you're already in `arch/aix` subdir. The `psutil` bit is only relevant in the `psutil` dir.